lib/game.rb

```diff
@@ -0,0 +1,104 @@
+class Game
+
+  attr_accessor :board, :player_1, :player_2, :winner
+
+  WIN_COMBINATIONS = [
+      [0,1,2],
+      [3,4,5],
+      [6,7,8],
+      [0,3,6],
+      [1,4,7],
+      [2,5,8],
+      [0,4,8],
+      [6,4,2]
+    ]
+
+  def initialize(player_1 = nil, player_2 = nil, board = nil) #assumes players and board are already initialized
+    if player_1 == nil
+      @player_1 = Human.new("X")
+    else
+      @player_1 = player_1
+    end
```

Comment:
You can simplify these assignments using the or operator (||).
```
@player_1 = player_1 || Human.new("X")
```
If the parameter `player_1` is set, it will be assigned to the instance variable `@player_1`. However, if the parameter is not passed in and defaults to `nil`, then the second half of the expression will be evaluated, setting the value of `@player_1` to `Human.new("X")`